tests/spec/createReactCustomElementType_spec.ts

```diff
@@ -0,0 +1,148 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2019
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import { act } from 'react-dom/test-utils';
+import EventManager from '../utils/event-manager';
+import createReactCustomElementType, {
+  booleanSerializer,
+  numberSerializer,
+  objectSerializer,
+} from '../../src/globals/wrappers/createReactCustomElementType';
+
+describe('React wrapper', function() {
+  let container;
+  const events = new EventManager();
+
+  beforeEach(function() {
+    container = document.createElement('div');
+    document.body.appendChild(container);
+  });
+
+  describe('Attribute mapping', function() {
+    it('Should use the property name the attribute name by default', function() {
```

Comment:
can this be reworded? it's a bit confusing

Reply:
Thanks @emyarod for your thoughts! How about adding "as"...? (...use the property name as the attribute name...)

Reply:
yes that would be better

Reply:
Thanks @emyarod! Updated.